### PR TITLE
[T3A5][W09-A3]Yim Chia Hui

### PR DESCRIPTION
**Pull request by sharkey1314**
Separated functionality of formatting from TextUi class to Formatter class

### CODE REVIEW

**Comment by sharkey1314** [comment]
Ready for review

File: src/seedu/addressbook/ui/TextUi.java [hunk 1]
```diff
@@ -83,37 +87,40 @@ public String getUserCommand() {
...
-        showToUser(
-                DIVIDER,
-                DIVIDER,
+        showToUser(WELCOME_MESSAGE,
                 MESSAGE_WELCOME,
                 version,
                 MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
-                storageFileInfo,
-                DIVIDER);
+                storageFileInfo);
```
Comment (hwkchia) [inline src/seedu/addressbook/ui/TextUi.java:101]: Some of these could still be pushed to the formatter to format the welcome message.  A little too much formatting mechanism here.

**Review by hwkchia — COMMENTED** [review]
Good first attempt in applying SRP.  May I suggest that you commit small changes and commit often?  That way it is easier for reviewer's to home in on typical problematic areas.

**Comment by hwkchia** [comment]
ACK by closing this PR.

**Pull request closed without merging**